store/models/common.go

```diff
@@ -21,21 +21,21 @@ const (
 	RunStatusInProgress = RunStatus("in progress")
```

Comment:
I also prefer the underscored status value. Let's add an underscore here for consistency?

Reply:
Added in #214